src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,63 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { useState } from 'react';
+
+interface TodoItemProps {
+  todo: Todo;
+}
+
+export const TodoItem: React.FC<TodoItemProps> = ({ todo }) => {
+  const { id, title, completed } = todo;
+  const [isPending] = useState(false);
```

Comment:
A state without a setter makes no sense.
In the future task you can create a state that stores an array of loading todo ids and depends on it show the loader